.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,12 @@
+{ 
+	"name": "Rust",
+        "image": "mcr.microsoft.com/devcontainers/rust:1-bullseye",
+	"features": {
```

Comment:
- I think the name field is unnecessary. e.g. https://github.com/devcontainers/features/blob/300814be97eaa5b7780886bd2eb37e96b523b3e7/.devcontainer.json
- Indentation should be consistent with either tab or space.
- `bookworm` may be better than `bullseye`. https://github.com/devcontainers/images/tree/main/src/rust
  (The reason the example I posted was `bullseye` is because it was copied from the old devcontainer.json that was used before `bookwarm` was released.)

Reply:
I think name is helpful once multiple dev-containers are used. For only one container it's obsolete. Happy to remove it

Reply:
changed to tabs and bookworm

Reply:
> I think name is helpful once multiple dev-containers are used. For only one container it's obsolete. Happy to remove it

In my experience, VS Code and GitHub display the repository name, so the name setting is almost irrelevant.

VS Code

![image](https://github.com/pola-rs/polars/assets/50911393/0509cbbf-0916-4ea3-9e1d-6660d1f1c23f)

GitHub

![image](https://github.com/pola-rs/polars/assets/50911393/0f42a19d-e2e5-40d1-907c-215e64080624)

I also don't think the name `Rust` is useful.
If multiple Rust-using projects all adopted the name `Rust`, I don't think it would be possible to tell them apart by name.

If you want to set a name, I recommend `"name": "${localWorkspaceFolderBasename}"`.